specification/adhybridhealthservice/resource-manager/Microsoft.ADHybridHealthService/stable/2014-01-01/ADHybridHealthService.json

```diff
@@ -0,0 +1,5772 @@
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/MetricSets"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/monitoringconfiguration": {
+      "patch": {
+        "tags": [ "Services" ],
+        "description": "Updates the service level monitoring configuration.",
+        "x-ms-examples": {
+          "services_updateMonitoringConfiguration": {
+            "$ref": "./examples/PatchMonitoringConfiguration.json"
+          }
+        },
+        "operationId": "services_updateMonitoringConfiguration",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "configurationSetting",
+            "in": "body",
+            "description": "The monitoring configuration to update",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Item"
+            }
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " "
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/monitoringconfigurations": {
+      "get": {
+        "tags": [ "Services" ],
+        "description": "Gets the service level monitoring configurations.",
+        "x-ms-examples": {
+          "services_listMonitoringConfigurations": {
+            "$ref": "./examples/MonitoringConfigurations.json"
+          }
+        },
+        "operationId": "services_listMonitoringConfigurations",
+        "parameters": [
+          {
+            "name": "serviceName",
+            "in": "path",
+            "description": "The name of the service.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": " ",
+            "schema": {
+              "$ref": "#/definitions/Items"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/providers/Microsoft.ADHybridHealthService/services/{serviceName}/reports/badpassword/details/user": {
+      "get": {
+        "tags": [ "Reports" ],
+        "description": "Gets the bad password login attempt report for an user",
+        "x-ms-examples": {
+          "services_listUserBadPasswordReport": {
+            "$ref": "./examples/BadPasswordDetails.json"
+          }
+        },
+        "operationId": "services_listUserBadPasswordReport",
+        "parameters": [
+          {
+            "name": "serv
```

Comment:
Looks like this should be x-ms-pageable

Reply:
I have it as pageable. See line 2039. Am I missing something here?

Reply:
@gamitra - sorry that comment somehow ended up on completely the wrong line - let me find the correct one...